data.js

```diff
@@ -820,6 +820,14 @@ module.exports = [
     url: "https://github.com/aravindbaskaran/simple-async",
     source: "https://raw.githubusercontent.com/aravindbaskaran/simple-async/master/asynciterator.js"
   },
+  {
+    name: "svelte-viewport-info",
+    github: "rozek/svelte-viewport-info",
+    tags: ["device-orientation","viewport-size"],
+    description: "informs about viewport size and orientation (not only in Svelte)",
+    url: "https://github.com/rozek/svelte-viewport-info",
+    source: "https://raw.githubusercontent.com/rozek/svelte-viewport-info/main/src/svelte-viewport-info.ts"
```

Comment:
From the readme - emphasis on "ready-to-use", which means you should not link to a TypeScript source file:

> The 'source' attribute must point to a single-file, or an array of files that can be concatenated to form your complete library in a ready-to-use form. The 'source' must not be minified (see below)